src/olympia/search/filters.py

```diff
@@ -163,7 +163,8 @@ def get_value(self):
                     ugettext('Return To AMO is currently disabled')
                 )
             try:
-                value = force_text(urlsafe_base64_decode(value[4:]))
+                value = force_text(
+                    urlsafe_base64_decode(force_text(value[4:])))
```

Comment:
Does the `_decode` version return a string too after 2.2? If so, let's mark it here too.

Reply:
No, it's always returned bytes, like the underlying https://docs.python.org/3/library/base64.html#base64.urlsafe_b64decode method, but django changed to require a string as input.  

I didn't bother adding a comment as it will actually fail with an exception in django2.2 if we remove the `force_text` wrapper from the input, unlike like `param = 'rta:%s' % force_text(urlsafe_base64_encode(force_bytes(addon.guid)))` lines where it wouldn't and we'd silently end up with `param = "rta:b'foo''`.  I can add a comment if you think it's clearer though?

Reply:
https://github.com/mozilla/addons-server/pull/10700/commits/8abb7668faa2b989928a4821c662a16be507169d